src/shell/services/instance.ts

```diff
@@ -57,6 +63,7 @@ export const instanceApi = createApi({
     "ItemWorkflowStatus",
     "WorkflowStatusLabels",
     "Groups",
+    "InstanceWorkflowStatusLabels",
```

Comment:
I already created a tag for this called `WorkflowStatusLabels`, don't forget to update your query's invalidatesTags

Reply:
updated.